arkouda/series.py

```diff
@@ -659,6 +660,69 @@ def to_pandas(self) -> pd.Series:
         else:
             return pd.Series(val.to_ndarray(), index=idx)
 
+    def to_markdown(self, mode="wt", index=True, tablefmt="grid", storage_options=None, **kwargs):
+        r"""
+        Print Series in Markdown-friendly format.
+
+        Parameters
+        ----------
+        mode : str, optional
+            Mode in which file is opened, "wt" by default.
+        index : bool, optional, default True
+            Add index (row) labels.
+        tablefmt: str = "grid"
+            Table format to call from tablulate:
+            https://pypi.org/project/tabulate/
+        storage_options: dict, optional
+            Extra options that make sense for a particular storage connection,
+            e.g. host, port, username, password, etc., if using a URL that will be parsed by fsspec,
+            e.g., starting “s3://”, “gcs://”.
+            An error will be raised if providing this argument with a non-fsspec URL.
+            See the fsspec and backend storage implementation docs for the set
+            of allowed keys and values.
+
+        **kwargs
+            These parameters will be passed to tabulate.
+
+        Note
+        ----
+        This function calls pandas.Series.to_markdown:
+        https://pandas.pydata.org/pandas-docs/version/1.2.4/reference/api/pandas.DataFrame.to_markdown.html
```

Comment:
we link to the dataframe version here, i think it's supposed to be the series version

Reply:
Fixed.